frontend/kesaseteli/employer/src/pages/login.tsx

```diff
@@ -1,23 +1,27 @@
-import useRouterClearQueryParams from 'kesaseteli/employer/hooks/useRouterClearQueryParams';
 import { GetStaticProps, NextPage } from 'next';
+import { useRouter } from 'next/router';
 import * as React from 'react';
+import Container from 'shared/components/container/Container';
 import withoutAuth from 'shared/components/hocs/withoutAuth';
 import Layout from 'shared/components/Layout';
+import useClearQueryParams from 'shared/hooks/useClearQueryParams';
 import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
 
 const Login: NextPage = () => {
-  const router = useRouterClearQueryParams();
+  useClearQueryParams();
   const {
     query: { logout, error, sessionExpired },
-  } = router;
+  } = useRouter();
   return (
-    <Layout headingText="Työnantajan liittymä">
-      {logout && <p>Olet kirjautunut ulos</p>}
-      {error && <p>Tapahtui tuntematon virhe. Kirjaudu uudelleen sisään.</p>}
-      {sessionExpired && (
-        <p>Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään.</p>
-      )}
-    </Layout>
+    <Container>
+      <Layout headingText="Työnantajan liittymä">
+        {logout && <p>Olet kirjautunut ulos</p>}
+        {error && <p>Tapahtui tuntematon virhe. Kirjaudu uudelleen sisään.</p>}
+        {sessionExpired && (
+          <p>Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään.</p>
+        )}
```

Comment:
Are these texts left out of the translation files on purpose?

Reply:
Yes this login page is very old and need to be finalized before release